src/DDoSX/CdnClient.php

```diff
@@ -13,7 +14,13 @@ class CdnClient extends BaseClient
      */
     protected $basePath = 'ddosx/';
 
-    protected $requestMap = [];
+    const CDN_MAP = [];
```

Comment:
++ for using named constants

Reply:
see below ln.33             `json_encode($this->friendlyToApi($cdn, $this->requestMap))`